src/interpreter.ts

```diff
@@ -55,7 +55,7 @@ const scan = (image: Image) => {
   const secondScan = cropAndScan(image, {
     x: width / 5,
     width: width / 5,
-    y: (height * 3) / 4,
+    y: (height * 3) / 4 - 1,
```

Comment:
because of rounding, this occasionally exceeds the bounds of the image, so make it squeeze in by 1 pixel.